src/pages/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import CarouselImg from '../components/Carousel/Carousel';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import GamesList from '../components/GamesList/GamesList';
+
+const Home = () => (
+  <main>
+    <CarouselImg />
+    <GamesList />
+  </main>
+);
```

Comment:
- [ ] At the moment, you are missing a very important component of the application as per project requirements. That is the filtering. A user should be able to filter data in your application following some parameter. It could be a category title or whatever depending on your specific application. At the moment a user only has to scroll to visit all the game types displayed on your homepage. You could implement a search box that can be used to filter the data.
![Screen Shot 2023-09-06 at 5 25 10 AM](https://github.com/JavierAybar/React_Games_Gallery/assets/62849872/bb9c8736-45b3-4fe0-86b1-f134a0c1b46b)